### PR TITLE
fix(quantic): changed ... to expand from a A tag to a button

### PR DESCRIPTION
**Pull request by drosenfeld-coveo**
[SVCC-2277](https://coveord.atlassian.net/browse/SVCC-2277)

Changed the clickable object from an A tag to a button as it does not redirect to another page. The mac screenreader now detects it as a expandable object

### CODE REVIEW

**Comment by nathanlb** [comment]
Could you post of screenshot or little screen capture video of what it looks like after the change?

**Comment by drosenfeld-coveo** [comment]
> Could you post a screenshot or little screen capture video of what it looks like after the change?

Uploading Screen Recording 2022-07-25 at 3.34.17 PM.mov…

**Review by aelfaleh — APPROVED** [review]

**Comment by erocheleau** [comment]
If you check my other PRs, I changed most of the <a> for <button> instead of lightning-button and what you need to make it look like before is this:

`class="slds-button"` this will give you the `variant="bare"` look, and you need to add the line-height: 1.5 to override the higher line-height from the salesforce buttons I think.

Don't know if overriding the height is possible if you use a lightning-button.

**Comment by lvu285** [comment]
I really like the part for aria-label, so it will have the consistent text across different screen reader

Probably out-of-scope, but I'm wonder if we can add something, to tell the user that user navigates to the "source-url-part" section? and totally how many "link" part we have right now? ("..." can count as 1 if it's in collapsed state)

**Review by nathanlb — APPROVED** [review]

File: packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <template for:each={foldedParents} for:item="parent">
         <div class="result-parenturi__container slds-grid slds-grid_vertical-align-center" key={parent.id}>
           <template if:true={parent.isFolded}>
-            <a onclick={expandParents}>...</a>
+            <button class="slds-button expand-parenturi__container" variant="base" label="..." title="Expand URI parts" aria-label="Expand URI parts" onclick={expandParents}>...</button>
```
Comment (drosenfeld-coveo) [inline packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html:10]: @acarpentier01 what do you think of the text "Expand URI parts"?

Comment (acarpentier01) [reply packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html:10]: I think "Expand URI" or "Expand path" would do the trick @drosenfeld-coveo. @jpmarceau Do you agree?

Comment (jpmarceau) [reply packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html:10]: I think so, unless by `part` you mean something technical, like a CSS part?

Comment (drosenfeld-coveo) [reply packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html:10]: I dont think the "part" has a specific meaning, I like "Expand URI"

Comment (acarpentier01) [reply packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html:10]: @erocheleau Just to confirm, is this is a URI or a breadcrumb? If it's a URI, then please go with "Expand URI" @drosenfeld-coveo.

**Review by btaillon — APPROVED** [review]

**Pull request merged**